Fixed a bug that the style palettes were not being loaded due an incorrect DOM event

cheb60 opened the pull request:
Fixed a bug that the custom style palettes were not being loaded on the front-end.  Swapped the page:loaded event for DOMContentLoaded because page:loaded is a custom event. Now the custom style palettes are being loaded correctly.

cheb60 committed:
Fixed a bug that the style palettes were not being loaded due a incorrect event.

Samuell1 commented:
@daftspunk said that it should use condition depending by turbo router `oc.useTurbo ? 'page:loaded' : 'DOMContentLoaded'`

Can you test it and i can merge this PR 💯

cheb60 committed:
Update scripts.htm

It indeed should use the condition depending by turbo router. Added compatibility by daftspunk request.

cheb60 commented:
@Samuell1 I added the condition as requested and it seems to work in both scenarios 👍 . Would be great if you can merge this PR.
